src/h3lib/lib/bbox.c

```diff
@@ -95,9 +95,10 @@ double _hexRadiusKm(H3Index h3Index) {
  *
  * @param bbox the bounding box to estimate the hexagon fill level
  * @param res the resolution of the H3 hexagons to fill the bounding box
- * @return the estimated number of hexagons to fill the bounding box
+ * @param out the estimated number of hexagons to fill the bounding box
+ * @return E_SUCCESS (0) on success, or another value otherwise.
  */
-int64_t bboxHexEstimate(const BBox *bbox, int res) {
+H3Error bboxHexEstimate(const BBox *bbox, int res, int64_t *out) {
     // Get the area of the pentagon as the maximally-distorted area possible
     H3Index pentagons[12] = {0};
     // TODO: Return error here
```

Comment:
Can we address this TODO as well? Might be unreachable, but we can now return the error if any